sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header title="Profile" />
+    </template>
+    <p class="category-header-medium">Student profile</p>
```

Comment:
it's a `H3` according to figma
![image](https://user-images.githubusercontent.com/77353155/231251565-08ba6743-3eed-4bd1-a5ce-6b9faf0ccce2.png)